src/Cuberite.vcxproj.user

```diff
@@ -24,4 +24,8 @@
     <LocalDebuggerWorkingDirectory>../Server</LocalDebuggerWorkingDirectory>
     <DebuggerFlavor>WindowsLocalDebugger</DebuggerFlavor>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
+    <LocalDebuggerWorkingDirectory>../Server</LocalDebuggerWorkingDirectory>
+    <DebuggerFlavor>WindowsLocalDebugger</DebuggerFlavor>
+  </PropertyGroup>
 </Project>
```

Comment:
Actually this file is rather needed - it sets up the initial project settings so that running in the debugger results in running the server in the correct folder.

Though it might be better to generate this file using cmake, next to the real project file (not assuming in-tree build), rather than store it in the git repo and assuming an in-tree build.